learning/tour-of-beam/frontend/lib/pages/tour/widgets/unit_content.dart

```diff
@@ -30,6 +32,76 @@ class UnitContentWidget extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
-    return MarkdownBody(data: unitContent.description);
+    final textTheme = Theme.of(context).textTheme;
+    final markdownStyle = MarkdownStyleSheet(
```

Comment:
Let's add `final MarkdownStyleSheet markdownStyleSheet` to `BeamThemeExtension`. This will simplify the dependencies on the theme below.

Reply:
Is it ok to add `flutter_markdown` dependency into `playground_components` for that?

Reply:
Yes. It is a lesser evil than building up dependencies on themes outside of `playground_components`.